_schema.graphql

```diff
@@ -4347,9 +4365,6 @@ type Mutation {
     input: CreateGeminiEntryForAssetInput!
   ): CreateGeminiEntryForAssetPayload
 
-  # Records an artwork view.
-  recordArtworkView(input: RecordArtworkViewInput!): RecordArtworkViewPayload
```

Comment:
I wasn't sure if this is intentional, but seems like this is gone now. 

cc @mzikherman